### PR TITLE
mgr/dashboard_v2: Updated reference documentation

### PR DESCRIPTION
**Pull request by LenzGr**
Updated the dashboard chapter in the manual to reflect the changes introduced by Dashboard v2 (e.g. username/password).

Updated description to include a high-level overview of the features available.

Signed-off-by: Lenz Grimmer <lgrimmer@suse.com>

### CODE REVIEW

**Review by Devp00l — DISMISSED** [review]

File: doc/mgr/dashboard.rst [hunk 1]
```diff
@@ -2,7 +2,29 @@ dashboard plugin
...
+following aspects of your Ceph cluster:
+
+* **Overall cluster health**: The overall cluster status, storage utilization
+  (e.g. number of objects, raw capacity, usage per pool), a list of pools and
+  their status and usage statistics, access to the cluster log file.
+* **Hosts**: A list of all hosts associated to the cluster, which services
+  are running and which version of Ceph is installed.
+* **Performance counters**: Display detailed statistics for each running service.
+* **Monitors**: List of all MONs, their quorum status, open sessions.
+* **Configuration Reference**: list all available configuration options, their
```
Comment (votdev) [inline doc/mgr/dashboard.rst]: s/list all/List all/

File: doc/mgr/dashboard.rst [hunk 2]
```diff
@@ -2,7 +2,29 @@ dashboard plugin
...
+* **OSDs**: A list of all OSDs, their status and usage statistics as well as
+  detailed information like attributes (OSD map), metadata, performance counters
+  and usage histograms for read/write operations.
+* **iSCSI**: List all hosts that run the TCMU runner service, list of all images
+  and their performance characteristics (read/write ops, traffic).
+* **RBD**: List of all RBD images and their properties (size, objects, features)
+  in a given pool. 
+* **CephFS**: List all active filesystem clients and associated pools, including
+  their usage statistics.
+* **Object Gateway**: list of all active object gateways and their performance
```
Comment (votdev) [inline doc/mgr/dashboard.rst]: s/list of/List of/

**Review by votdev — DISMISSED** [review]

File: doc/mgr/dashboard.rst (hunk 1 above)
Comment (LenzGr) [reply doc/mgr/dashboard.rst]: Fixed, thanks!

File: doc/mgr/dashboard.rst (hunk 2 above)
Comment (LenzGr) [reply doc/mgr/dashboard.rst]: Fixed.

**Review by Devp00l — APPROVED** [review]

**Review by votdev — APPROVED** [review]

**Pull request merged**